.googleConfig.example.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
I think the `.googleConfig.example.js` wouldn't be needed once PR #41 is merged

Reply:
> 

fixed